package.json

```diff
@@ -23,6 +23,7 @@
     "lib",
     "cli.js"
   ],
+  "typings": "./d.ts/index.d.ts",
```

Comment:
@jbucaran I don't know about other editors but i am pretty sure that vscode and webstorm can do that.